src/Sfa.Das.Sas.Indexer.ApplicationServices/Lars/Settings/LarsIndexSettings.cs

```diff
@@ -14,5 +14,6 @@ public class LarsIndexSettings : IIndexSettings<IMaintainLarsIndex>
         public string StandardProviderDocumentType => ConfigurationManager.AppSettings["StandardProviderDocumentType"];
 
         public string FrameworkProviderDocumentType => ConfigurationManager.AppSettings["FrameworkProviderDocumentType"];
+        public string ProviderExclusionList => ConfigurationManager.AppSettings["ProviderExclusionList"];
```

Comment:
These settings need to start coming from CloudConfigurationManager.GetSetting("ProviderExclusionList")